ansible/inventory/env/group_vars/all.yml

```diff
@@ -655,5 +655,6 @@ enc_n_keys: 50 # Total number of keys required for encryption
 enc_n_reserved_keys: 15 # Number of keys which app reserves for usage
 kafka_topics_instruction: "{{env_name}}.coursebatch.job.request"
 kafka_urls: "{{groups['processing-cluster-kafka']|join(':9092,')}}:9092"
+kafka_topics_certificate_instruction: "{{env_name}}.certificate.job.request"
```

Comment:
change it with below.
kafka_topics_certificate_instruction: "{{env_name}}.coursebatch.certificate.request"